build.sc

```diff
@@ -92,9 +92,8 @@ object Deps {
   )
   val asciidoctorj = ivy"org.asciidoctor:asciidoctorj:2.4.3"
   val bloopConfig = ivy"ch.epfl.scala::bloop-config:1.4.13"
-  val coursier = ivy"io.get-coursier::coursier:2.0.16-200-ge888c6dea"
+  val coursier = ivy"io.get-coursier::coursier:2.1.0-M2"
   val coursierReducedDeps = coursier.exclude(
-    "com.lihaoyi" -> "utest",
```

Comment:
Is this no longer in the transitive dependency set, or is it mandatory now?

Reply:
It's not a dependency any more, it had ended up there by mistake…

Reply:
Ok. Nice you fixed it.